source/upgrade/important-upgrade-notes.rst

```diff
@@ -15,11 +15,30 @@ Important Upgrade Notes
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| v7.0                                               | MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in FileInfo table. For MySQL, it takes around   |
+|                                                    | 19 seconds for a table of size 700,000 rows. The time required for PostgreSQL is negligible. The testing was performed on a machine with specifications of       |
+|                                                    | ``CPU - Intel i7 6-cores @ 2.6 GHz`` and ``Memory - 16 GB``.                                                                                                     |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | When a new configuration setting via **System Console > Experimental > Features > Enable App Bar** is enabled, all channel header icons registered by plugins    |
+|                                                    | will be moved to the new Apps Bar, even if they do not explicitly use the new registry function to render a component there. The setting for Apps Bar defaults   |
+|                                                    | to ``false`` for self-hosted deployments.                                                                                                                        |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | The value of ``ServiceSettings.TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions.         |
+|                                                    | Customers are requested to check for these values in their config and set them to nil if necessary. See more details                                             |
+|                                                    | `here <https://docs.mattermost.com/configure/configuration-settings.html#trusted-proxy-ip-header>`_.                                                             |            
++----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Collapsed Reply Threads (link to docs) is now generally available and enabled by default for new Mattermost servers. For servers upgrading to v7.0 and later, please reference [this article](link to KB article from @cwarnermm) for more information and guidance for enabling the feature.

Reply:
The URL of the new KB article on CRT GA will be: https://mattermost.zendesk.com/knowledge/articles/6880701948564/en-us?brand_id=327479

Note that this KB article is currently in draft and not accessible from the KB site. It will be published live on June 14th.